apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -316,25 +316,25 @@ function enqueue_coblocks_gallery_scripts() {
 add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_coblocks_gallery_scripts' );
 
 /**
- * Load Homepage Posts block.
+ * Load Blog Posts block.
  */
-function load_homepage_posts_block() {
+function load_blog_posts_block() {
 	$disable_block = (
-		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_option( 'active_plugins', [] ), true ) ||
-		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_site_option( 'active_sitewide_plugins', [] ), true )
+		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_option( 'active_plugins', array() ), true ) ||
+		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_site_option( 'active_sitewide_plugins', array() ), true )
```

Comment:
😢

Reply:
I asked about this as well. 😅

Reply:
I'm 99% sure this was done for me by the precommit hook. I haven't touched the syntax and I'd use `[]` on my own